geode-core/src/integrationTest/java/org/apache/geode/distributed/ServerLauncherLocalIntegrationTest.java

```diff
@@ -261,17 +281,20 @@ public void statusForDisableDefaultServerHasEmptyPort() {
   public void statusWithPidReturnsOnlineWithDetails() throws UnknownHostException {
     givenRunningServer();
 
-    ServerState serverState = new Builder().setPid(localPid).build().status();
+    ServerState serverState = new Builder()
+        .setPid(localPid)
+        .build()
+        .status();
 
-    assertThat(serverState.getStatus()).isEqualTo(ONLINE);
     assertThat(serverState.getClasspath()).isEqualTo(getClassPath());
     assertThat(serverState.getGemFireVersion()).isEqualTo(GemFireVersion.getGemFireVersion());
-    assertThat(serverState.getHost()).isEqualTo(InetAddress.getLocalHost().getCanonicalHostName());
+    assertThat(serverState.getHost()).isEqualTo(getLocalHost().getCanonicalHostName());
```

Comment:
Without going to deep into code, but are we sure this is will always result in localhost? What would this value be in a multi-homed environment? Does this not matter for this test?

Reply:
You're right. I don't know. However, I have already committed other tests that fail on a Mac with non-default /etc/hosts and those tests are passing in CI, on various people's macs, and I haven't heard of any multi-homed issues. If you see a failure in a multi-homed environment, then I recommend filing a Jira ticket.